source/features/more-dropdown.tsx

```diff
@@ -43,7 +43,7 @@ function onlyShowInDropdown(id: string): void {
 		return;
 	}
 
-	tabItem!.parentElement!.remove();
+	tabItem!.remove();
```

Comment:
Seems like the `[data-tab-item$="${id}"]` selector now points to the tab anchor/link.
![image](https://user-images.githubusercontent.com/10238474/112874906-2b5a5300-90cc-11eb-80c6-79c4a7397ba0.png)